yt/visualization/plot_window.py

```diff
@@ -2466,6 +2463,13 @@ def __init__(
                 "currently supported geometries:"
                 f" {self._supported_geometries!r}"
             )
+
+        if depth is None:
+            # off-axis projection, depth not specified
+            # -> set 'large enough' depth using half the box diagonal + margin
+            depth = np.sqrt(np.sum(ds.domain_width.in_units("code_length") ** 2)) * 1.02
```

Comment:
```suggestion
            depth = np.linalg.norm(ds.domain_width.in_units("code_length")) * 1.02
```
going to let tests run before making this change